superset/charts/dao.py

```diff
@@ -27,9 +27,6 @@
 from superset.models.slice import Slice
 from superset.utils.core import get_user_id
 
-if TYPE_CHECKING:
-    from superset.connectors.base.models import BaseDatasource
```

Comment:
This was never used. I'm surprised Pylint didn't recognize it, but maybe it's not well versed with the `TYPE_CHECKING` construct and thus it felt it was used.